src/applications/find-forms/components/FindVaForms.jsx

```diff
@@ -13,111 +20,130 @@ const onFeaturedContentClick = header => () => {
-      </li>
-      <li className="featured-content-list-item vads-u-background-color--primary-alt-lightest  vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0 vads-u-display--flex vads-u-flex-direction--column">
-        <h3 className="vads-u-font-size--base vads-u-margin--0">
-          Apply for VA health care
-        </h3>
-        <hr
-          aria-hidden="true"
-          role="presentation"
-          className="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary"
-        />
-        <p className="va-nav-linkslist-description">Equal to VA Form 10-10EZ</p>
-        <a
-          className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
-          href="/health-care/apply/application/"
-          onClick={onFeaturedContentClick('Apply for VA health care benefits')}
-        >
-          <span>
-            Apply online
-            <span className="vads-u-visibility--screen-reader">
-              about applying for VA health care benefits
+          </a>
+        </li>
+        <li className="featured-content-list-item vads-u-background-color--primary-alt-lightest  vads-u-padding-y--1p5 vads-u-padding-x--1p5 vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0 vads-u-display--flex vads-u-flex-direction--column">
+          <h3 className="vads-u-font-size--base vads-u-margin--0">
+            Apply for VA health care
+          </h3>
+          <hr
+            aria-hidden="true"
+            role="presentation"
+            className="featured-content-hr vads-u-margin-y--1p5 vads-u-border-color--primary"
+          />
+          <p className="va-nav-linkslist-description">
+            Equal to VA Form 10-10EZ
+          </p>
+          <a
+            className="vads-u-display--block vads-u-padding-top--1 vads-u-text-decoration--none"
+            href="/health-care/apply/application/"
+            onClick={onFeaturedContentClick(
+              'Apply for VA health care benefits',
+            )}
+          >
+            <span>
+              Apply online
+              <span className="vads-u-visibility--screen-reader">
+                about applying for VA health care benefits
+              </span>
+              <i
+                aria-hidden="true"
+                className="fa fa-chevron-right vads-facility-hub-cta-arrow"
+                role="presentation"
+              />
             </span>
-            <i
-              aria-hidden="true"
-              className="fa fa-chevron-right vads-facility-hub-cta-arrow"
-              role="presentation"
-            />
-          </span>
-        </a>
-      </li>
-    </ul>
-  </>
-);
+          </a>
+        </li>
+      </ul>
+    </>
+  );
+};
+
+FindVaForms.propTypes = {
+  showPdfWarningBanner: PropTypes.bool,
+};
+
+const mapStateToProps = state => ({
+  showPdfWarningBanner: toggleValues(state)[
+    FEATURE_FLAG_NAMES.pdfWarningBanner || false
+  ],
```

Comment:
It looks like the fallback (`... || false`) might be in the wrong place here. I think it should be outside the brackets. 

That said, it might be even better to define a default value for this property on the component definition and then just leave this map function as a possibly undefined value.
```
export const FindVaForms = ({ showPdfWarningBanner = false }) => {
```